### PR TITLE
Update QUEST_LV_0100.tsv

### PR DESCRIPTION
**Pull request by Sunaries**
Very minor fix.

### CODE REVIEW

File: QUEST_LV_0100.tsv [hunk 1]
```diff
@@ -641,7 +641,7 @@ QUEST_LV_0100_20150317_000640	Coben
 QUEST_LV_0100_20150317_000641	I am looking for my brother.{nl}If you could help me, I may give you some rewards.
 QUEST_LV_0100_20150317_000642	He was really fanstastic.{nl}A person like me can not imagine.
 QUEST_LV_0100_20150317_000643	Something is between the pages of diary.{nl}Blueprint.. Brother got something again.
-QUEST_LV_0100_20150317_000644	I came here to look for my brother, but I am not brave like my brother.{nl}I want to you to take a look at the gravestone's writing on behalf of me.
+QUEST_LV_0100_20150317_000644	I came here to look for my brother, but I am not brave like my brother.{nl}I want you to take a look at the gravestone's writing on behalf of me.
```
Comment (Jasdemi) [inline QUEST_LV_0100.tsv]: "..., but I am not as brave as him." Sounds better imo.

Comment (ghost) [inline QUEST_LV_0100.tsv]: Also, "...on my behalf."

**Comment by Sunaries** [comment]
:+1: 

Good to merge.

**Comment by imcgames** [comment]
Thank you :smile:

**Pull request merged**